src/contracts/deposit/interfaces/event.interface.ts

```diff
@@ -7,6 +7,9 @@ export interface DepositEvent {
   blockNumber: number;
   blockHash: string;
   logIndex: number;
+  index: string;
+  depositCount: number;
```

Comment:
is it a unique number of event in a deposit contract?

Reply:
depositCount is an incremental variable that is incremented for each deposit.